src/Modules/ModParser.lua

```diff
@@ -3370,6 +3369,15 @@ local specialModList = {
 	["voltaxic burst deals (%d+)%% increased damage per ([%d%.]+) seconds of duration"] = function(num, _) return { mod("VoltaxicDurationIncDamage", "INC", num) } end,
 	["earthquake deals (%d+)%% increased damage per ([%d%.]+) seconds duration"] = function(num, _) return { mod("EarthquakeDurationIncDamage", "INC", num) } end,
 	["consecrated ground from holy flame totem applies (%d+)%% increased damage taken to enemies"] = function(num) return { mod("EnemyModifier", "LIST", { mod = mod("DamageTakenConsecratedGround", "INC", num, { type = "Condition", var = "OnConsecratedGround" }) }) } end,
+	["consecrated ground from purifying flame applies (%d+)%% increased damage taken to enemies"] = function(num) return { mod("EnemyModifier", "LIST", { mod = mod("DamageTakenConsecratedGround", "INC", num, { type = "Condition", var = "OnConsecratedGround" }) }) } end,
```

Comment:
_Technically_ this might not apply to all consecrated ground like you have it here, so you could make this conditional on having Purifying Flame, but I think it's fine.  The alt quality Purifying Flame has the same problem anyway

Reply:
Oh yeah I should make it conditional. I made it the same way holy flame totem was done above. I’ll fix both of them